### ISSUE TITLE
Intl example in the examples/ folder

### ISSUE THREAD

**Issue by jimmyff** [dart-lang/angular#1167]
It would be good to have an example of using `package:intl` the examples folder.

I've put my translation string functions in a LocaleService which I expose to each template. I'm not sure if this is good or bad practice.

**Comment by matanlurey** [comment]
@leonsenft Do you want to take this as part of your Intl exploration?

**Comment by leonsenft** [comment]
Yeah as part of whatever intl solution we land on, I'll definitely add an example to go along with the feature.

**Comment by gedw99** [comment]
It great be good is the components supported it too.
Here is what i mean:
Code: https://github.com/material-components/material-components-web
Demo: https://material-components-web.appspot.com/

RTL and LTR switching so you can see it working.
Language switching.
